tcf_website/templates/search/search.html

```diff
@@ -72,10 +72,43 @@ <h5><a href="{% url 'department' dept_id=dept_data.dept_id %}">{{ dept }} / {{ d
                               </a>
                             </li>
                         {% endfor %}
-                    </ul>
+                    </ul>-->
+                    <h5><a href="{% url 'department' dept_id=dept_data.dept_id %}">{{ dept }} / {{ dept_data.subdept_name }}</a></h5>
+                      {% for c in dept_data.courses %}
+                        {% if c != dept_data.courses.0 %}
+                        {% endif %}
+                        <div class="list-group-item list-group-item-light text-dark p-2">
+                          <div class="row">
+                            <div class = "col-md-11 align-self-center">
+                                <a href="{% url 'course' mnemonic=c.mnemonic.split.0 course_number=c.number %}">
+                                  <div class = "row g-0 ">
```

Comment:
In general try limiting these extra spaces in attribute names/values like with "class = ..." instead of "class=...."; they're not wrong, just tend to be superfluous